app/workers/mails/member_job.rb

```diff
@@ -84,8 +78,4 @@ def every_group_user_member(member, &block)
       .includes(:project, :principal, :roles, :member_roles)
       .each(&block)
   end
-
-  def sending_disabled?(setting, message)
-    message.blank? && !Setting.notified_events.include?("membership_#{setting}")
-  end
```

Comment:
The now user specific setting for whether or not to be notified still needs to be applied here unfortunately, as the members do not yet use the notifications pipeline.